src/test/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SeparatorWrapCheckTest.java

```diff
@@ -91,7 +91,7 @@ public void testInvalidOption() throws Exception {
         try {
             final String[] expected = CommonUtils.EMPTY_STRING_ARRAY;
 
-            verify(checkConfig, getPath("InputSeparatorWrap.java"), expected);
+            verify(checkConfig, getPath("InputSeparatorWrapForInvalidOption.java"), expected);
```

Comment:
InputSeparatorWrapForInvalidOption can contain an empty class. The main purpose of the test is to check that passing ivalid option will throw exception.